projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -747,6 +750,15 @@ public function should_link_to_wp_admin() {
 		return get_user_option( 'jetpack_admin_menu_link_destination' );
 	}
 
+	/**
+	 * Returns the first available upsell nudge.
+	 *
+	 * @return array
+	 */
+	public function get_upsell_nudge() {
+		return array();
+	}
+
```

Comment:
Can we keep this in the `Admin_Menu` class since this method is only needed in there? I'd also add a comment explaining the method returns nothing because it should be implemented in the child classes.

Reply:
Sure! Done in https://github.com/Automattic/jetpack/pull/24304/commits/61f2ef9fc895b4117947516b29760d92c346ebd3